sim/dex-data.js

```diff
@@ -142,11 +142,27 @@ class Format extends Effect {
 		 */
 		this.ruleset = this.ruleset;
 		/**
+		 * Base list of rule names as specified in "./config/formats.js".
+		 * Used in a custom format to correctly display the altered ruleset.
+		 * @type {string[]}
+		 */
+		this.baseRuleset = this.baseRuleset;
```

Comment:
I'm not sure what you mean. <code>this.ruleset</code> and <code>this.banlist</code> were previously documented the same way.